public/data/roshanmhatre.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Roshan Mhatre",
+    "bio": "🤎Web Dev and Open-Source Enthusiast",
+    "avatar": "https://github.com/RoshanMhatre.png",
+    "links": [
+      {
+        "name": "My GitHub",
+        "url": "https://github.com/RoshanMhatre",
+        "icon": "github"
+      },
+      {
+        "name": "My Twitter",
+        "url": "https://twitter.com/RoshanMhatre",
+        "icon": "twitter"
+      }
+    ]
+  }
+  
```

Comment:
GitHub recommends an empty blank line at the end of each file, displayed with the red icon

Reply:
Ok Great! Thanks for letting me know, will keep in mind next time.